modules/administration-guide/pages/installing-che-in-an-oidc-unsupported-kubernetes.adoc

```diff
@@ -0,0 +1,220 @@
+:_content-type: PROCEDURE
+:navtitle: Installing {prod-short} in an OIDC Unsupported {kubernetes}
+:description: Installing {prod-short} in {kubernetes} that doesn't support OIDC yet.
+:keywords: installing-che-in-an-unsupported-{kubernetes}
+:page-aliases: installation-guide:installing-che-in-oidc-unsupported-kubernetes
+
+[id="installing-{prod-id-short}-in-odic-unsupported-kubernetes"]
+= Installing {prod-short} in OIDC Unsupported {kubernetes} Cluster
+
+You can deploy {prod-short} into a {kubernetes} cluster that doesn't support OIDC.
+
+WARNING: vCluster is a third party tool. See link:https://www.vcluster.com/docs/what-are-virtual-clusters[Introduction to vCluster]
+
+.Prerequisites
+
+* Helm CLI. See link:https://helm.sh/docs/intro/install/[Installing Helm].
+
+* vCluster CLI. See link:https://www.vcluster.com/docs/getting-started/setup[Installing vCluster CLI].
+
+* `{orch-cli}` stable release with `oidc-login` plugin. See link:https://kubernetes.io/docs/tasks/tools/#kubectl[Installing `{orch-cli}`], link:https://github.com/int128/kubelogin[Installing `kubelogin`].
+
+* `{prod-cli}`. See xref:installing-the-chectl-management-tool.adoc[].
+
+* External OIDC Server. Example: Keycloak. 
```

Comment:
It would be nice if your example includes installing Keycloak as well.
I will be helpful for users who are not familiar with that.
For instance https://eclipse.dev/che/docs/stable/administration-guide/installing-che-on-minikube-keycloak-oidc/

It is possible to deploy external OIDC server on the same cluster?